Components/LandingPage/WhyChooseUs.tsx

```diff
@@ -31,14 +28,44 @@ const WhyChooseUs = () => {
   );
 };
 
-const InfoCard = ({ title, description}: any) => {
+const InfoCard = ({ title, description }: any) => {
```

Comment:
`any` is not the right type. Please use the type what it will return and the type for `title` and `description` should also be there.